### DISCUSSION TITLE
ckb 0.114.0 (0c8711e 2024-02-29)

### DISCUSSION THREAD

**Discussion by doitian** [nervosnetwork/ckb#4370] [Announcements]
### Highlights

Packaged ckb-cli v1.7.0 and updated ckb-vm to v0.24.8.

### Compatibility Table

| Network | Consensus Version        | Minimal CKB Version |
|---------|------------------------- | ------------------- |
| dev     | ckb2023 since epoch 0    |                     |
| testnet | ckb2021 since epoch 3113 | v0.101.0            |
| mainnet | ckb2021 since epoch 5414 | v0.103.0            |

See [how to connect to various chains](https://github.com/nervosnetwork/ckb/wiki/Chains) in Wiki.

### Downloads

| OS                | Arch  | Runtime Dependencies | Package                 | Sign                |
| ----------------- | ----- | -------------------- | ----------------------- | ------------------- |
| macOS             | x64   | macOS 10.15 or above | [zip][macos-zip]        | [PGP][macos-pgp]    |
| macOS  (Portable) | x64   | macOS 10.15 or above | [zip][macos-p-zip]      | [PGP][macos-p-pgp]  |
| macOS             | ARM64 | macOS 10.15 or above | [zip][macosa-zip]       | [PGP][macosa-pgp]   |
| macOS  (Portable) | ARM64 | macOS 10.15 or above | [zip][macosa-p-zip]     | [PGP][macosa-p-pgp] |
| Linux             | x64   | glibc, libstdc++     | [tarball][linux-zip]    | [PGP][linux-pgp]    |
| Linux  (Portable) | x64   | glibc, libstdc++     | [tarball][linux-p-zip]  | [PGP][linux-p-pgp]  |
| CentOS            | x64   | glibc, libstdc++     | [tarball][centos-zip]   | [PGP][centos-pgp]   |
| CentOS (Portable) | x64   | glibc, libstdc++     | [tarball][centos-p-zip] | [PGP][centos-p-pgp] |
| Windows           | x64   | VC++ Redistributable | [zip][windows-zip]      | [PGP][windows-pgp]  |
| Linux             | ARM64 | glibc, libstdc++     | [tarball][arm-zip]      | [PGP][arm-pgp]      |
| Docker            |       |                      | [dockerhub][]           |                     |

The `.asc` files are signatures. It is wise and more secure to check out for the files [integrity](https://github.com/nervosnetwork/ckb/blob/develop/docs/integrity-check.md).

Try the portable variants if the binaries complains about illegal instructions.

[macos-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-apple-darwin.zip
[macos-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-apple-darwin.zip.asc
[macos-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-apple-darwin-portable.zip
[macos-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-apple-darwin-portable.zip.asc
[macosa-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_aarch64-apple-darwin.zip
[macosa-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_aarch64-apple-darwin.zip.asc
[macosa-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_aarch64-apple-darwin-portable.zip
[macosa-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_aarch64-apple-darwin-portable.zip.asc
[linux-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-linux-gnu.tar.gz
[linux-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-linux-gnu.tar.gz.asc
[linux-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-linux-gnu-portable.tar.gz
[linux-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-linux-gnu-portable.tar.gz.asc
[centos-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-centos-gnu.tar.gz
[centos-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-centos-gnu.tar.gz.asc
[centos-p-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-centos-gnu-portable.tar.gz
[centos-p-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-unknown-centos-gnu-portable.tar.gz.asc
[windows-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-pc-windows-msvc.zip
[windows-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_x86_64-pc-windows-msvc.zip.asc
[arm-zip]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_aarch64-unknown-linux-gnu.tar.gz
[arm-pgp]: https://github.com/nervosnetwork/ckb/releases/download/v0.114.0/ckb_v0.114.0_aarch64-unknown-linux-gnu.tar.gz.asc
[dockerhub]: https://hub.docker.com/r/nervos/ckb/tags

#### Features

* Package ckb-cli v1.7.0
* #4361: Update ckb-vm to v0.24.8 (@mohanson)
* #4358: Add `--include-background` to include background migrations in migrate subcommand (@chenyukang)

#### Bug Fixes

* #4349: A peer send manlformed tx with large cycles will also be banned (@chenyukang)

<hr /><em>This discussion was created from the release <a href='https://github.com/nervosnetwork/ckb/releases/tag/v0.114.0'>ckb 0.114.0 (0c8711e 2024-02-29)</a>.</em>

**Comment by joshyates1980** [comment]
This is different when I execute **ckb run --indexer**

![image](https://github.com/nervosnetwork/ckb/assets/7255545/d9c34748-1cb4-4b4f-bc3b-f247c4da25f6)

**Reply by chenyukang** [reply]
You mean there is a progress bar?
There is a background migration for 114.0, the progress bar is from migration, you can ignore it.
